enterprise/backend/src/metabase_enterprise/dependencies/api.clj

```diff
@@ -359,34 +452,59 @@
                        (apply merge-with +)))
                 children-map)))
 
-(defn- expanded-nodes [downstream-graph nodes]
+(defn- calc-errors [nodes-by-type]
```

Comment:
When I see this it makes me think it returns calculation errors or something, but I know that it doesn't, which makes me think maybe it needs a better name